ipa-core/src/ff/boolean_array.rs

```diff
@@ -318,6 +318,17 @@ boolean_array_impl!(
     )
 );
 
+//impl BA4
+boolean_array_impl!(
+    boolean_array_4,
+    BA4,
+    4,
+    1,
+    bitarr ! ( const u8, Lsb0;
+        1, 0, 0, 0
+    )
```

Comment:
I find the formatting here disturbing.  Is there any reason that we can't have this be "[1, 0, 0, 0]" with the common part (`bitarr![const u8, Lsb0; ...]`) supplied by the macro?

Reply:
I can't figure out how to do this.

The last thing is supposed to be an "expression". I tried an array, a tuple, and neither of them can be used in that macro.  I'm going to leave this as is for now.